Sources/AudioKit/MIDI/MIDI+Sending.swift

```diff
@@ -352,27 +287,31 @@ extension MIDI {
     ///   - noteNumber: MIDI Note Number
     ///   - velocity: MIDI Velocity
     ///   - channel: MIDI Channel (default: 0)
+    ///   - time: MIDI Timestamp (default: mach_absolute_time())
     public func sendNoteOnMessage(noteNumber: MIDINoteNumber,
                                   velocity: MIDIVelocity,
                                   channel: MIDIChannel = 0,
+                                  time: MIDITimeStamp = MIDITimeStamp(mach_absolute_time()),
                                   endpointsUIDs: [MIDIUniqueID]? = nil,
                                   virtualOutputPorts: [MIDIPortRef]? = nil) {
         let noteCommand: MIDIByte = noteOnByte + channel
         let message: [MIDIByte] = [noteCommand, noteNumber, velocity]
-        self.sendMessage(message, endpointsUIDs: endpointsUIDs, virtualOutputPorts: virtualOutputPorts)
+        self.sendMessage(message, time: time, endpointsUIDs: endpointsUIDs, virtualOutputPorts: virtualOutputPorts)
     }
 
     /// Send a Note Off Message
     /// - Parameters:
     ///   - noteNumber: MIDI Note Number
     ///   - channel: MIDI Channel (default: 0)
+    ///   - time: MIDI Timestamp (default: mach_absolute_time())
```

Comment:
May want to add a note to the inline doc here that `time` should never be a value of 0